### PR TITLE
Add popup pull request

### PR DESCRIPTION
**Pull request by alicemirigo92**
Use basic JavaScript syntax.
Use JavaScript to manipulate DOM elements.
Use JavaScript events.
Understand how to parse a Figma design to create a UI.
Use objects to store and access data.

### CODE REVIEW

File: modalstyle.css [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+    height: 20px;
+    top: 5px;
+    right: 15px;
+    cursor: pointer;
+  }
+
+  #arrow {
+    width: 20px;
+    height: 20px;
+  }
```
Comment (Tufahel) [inline modalstyle.css]: - [ ] Kindly fix the position and color of the `name` and `arrow` inside the `See Live` button according to the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
***Figma Design***
Screenshot from 2023-01-06 04-46-00
***Your Design***
Screenshot from 2023-01-06 04-46-49

File: modalstyle.css [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+    background-color: #1a2236;
+    color: white;
+  }
+
+  .live-btn {
+    width: 246px;
+    height: 51px;
+    border: 1px solid #1a2236;
+    gap: 10px;
+  }
```
Comment (Tufahel) [inline modalstyle.css:64]: - Great Work on implementing `hover` on the buttons. :+1: 

- [ ] Kindly fix the position and the color of the `name` and `GitHub` link inside the `See Source` button.
***Figma Design***
Screenshot from 2023-01-06 05-15-17
***Your Design***
Screenshot from 2023-01-06 04-54-00

File: modalstyle.css [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+    align-items: center;
+    padding: 3%;
+    margin: 5% auto;
+  }
+
+  .modal-links {
+    display: inline-block;
+    gap: 10px;
+    margin-bottom: 20px;
+  }
```
Comment (Tufahel) [inline modalstyle.css:32]: - [ ] Kindly follow the given [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9), to `style the fonts, the position of the text and for margin/padding`.
***Figma Design***
Screenshot from 2023-01-06 04-57-59
***Your Design***
Screenshot from 2023-01-06 04-59-29

File: index.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+const btns = document.querySelectorAll('.btn');
+btns.forEach((b) => {
+  b.addEventListener('click', () => {
+    const parent = b.parentNode;
+    const div = document.createElement('div');
+    div.className = 'popup';
+    div.innerHTML = `<div class="modalbox">
+        <img id="cross" src="./images/ic_cross.svg" alt="ic_cross">
+        <p>${projects[b.id].title}</p>
```
Comment (Tufahel) [inline index.js:66]: - [ ] As the project name is a title, it is recommended to use `<hx></hx>` for titles, not `<p></p>` tag, to increase HTML accessibility.
  - Such as you can consider using `<h2></h2>` or `<h3></h3>` tag.

- [ ] Kindly follow the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9) to style the title.
***Your Design***
Screenshot from 2023-01-06 05-08-46
***Figma Design***
Screenshot from 2023-01-06 05-07-58

File: modalstyle.css [hunk 5]
```diff
@@ -0,0 +1,195 @@
+@media (max-width: 768px) {
```
Comment (Tufahel) [inline modalstyle.css:1]: - [ ] Kindly fix the `Style` of the modal of the `Mobile` version. Your project style looks the same for the `Mobile` and `Desktop` versions. But in [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9), styles are different in both cases.
***Your Design***
Screenshot from 2023-01-06 05-10-02
***Figma Design***
Screenshot from 2023-01-06 05-14-31

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @alicemirigo92 @Zeph180 :wave: 

Good Job so far. :+1:  
There are some issues you need to fix before you move to the next project.

## Highlights 🌟

Alt Text

✔️ Linters checks are passing
✔️ Use GitHub Flow
✔️ Meaningful commit messages.
✔️ Correctly implemented an array of objects to store JavaScript objects.
✔️ Correctly implemented Modal.

## Required Changes ♻️

- [ ] Kindly implement `Previous project` and `Next project` at the bottom of the popup window according to the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) for `Desktop` version.
***Figma Design***
Screenshot from 2023-01-06 05-29-38

_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+    source: '#',
+  },
+};
+
+const btns = document.querySelectorAll('.btn');
+btns.forEach((b) => {
+  b.addEventListener('click', () => {
+    const parent = b.parentNode;
+    const div = document.createElement('div');
+    div.className = 'popup';
```
Comment (brainconnect93) [inline index.js:63]: - [ ] As requested by the previous reviewer, I would kindly suggest you follow the Figma template as much as possible for your modal pop-up design

Below 👇  is a video explanation of what you need to work on

[screen-capture (3).webm](https://user-images.githubusercontent.com/78634841/211008759-03a755de-a4ef-4482-a23c-7341316df24b.webm)

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @alicemirigo92, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by alicemirigo92** [comment]
@brainconnect93 Done

**Review by iambenkis — APPROVED** [review]
### Hi  @alicemirigo92    , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


cr-approve

## Highlights

GitHub review feature :heavy_check_mark:
Inline comment :heavy_check_mark: 
The comments are given in a friendly and polite manner :heavy_check_mark: 
Explain why changes were suggested :heavy_check_mark: 

## Optional 
**_Every comment with the [**OPTIONAL**] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@iambenkis** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**